dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/transport/AbstractClient.java

```diff
@@ -152,6 +152,8 @@ private synchronized void initConnectStatusCheckCommand() {
                 @Override
                 public void run() {
                     try {
+                        if (cancelFutureIfOffline()) return;
```

Comment:
Can be considered plus `{}`  to Increase readability

Reply:
done